### PR TITLE
docs: add project vision

### PR DESCRIPTION
**Pull request by jhmarina**
# Describe your changes

This PR improves the information on the main docs page. It includes content from the [internal project page](https://confluence.suse.com/display/RES/S3+gateway+project), explaining the project's vision, why we're doing it & use cases.

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.
- [ ] CHANGELOG.md has been updated should there be relevant changes in this PR.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,29 +1,51 @@
...
+
+### Why are we doing it?
+
+We have identified a need for making cluster data backups easily available for
+apps that don't require petabyte-scale storage.
+
+### Features
+
+- An intuitive UI
+- Lightweight filesystem to store metadata
```
Comment (giubacc) [inline docs/index.md]: I would rephrase with something like:
_sqlite database for a lightweight management of  s3-objects metadata_

**Review by m-ildefons — CHANGES_REQUESTED** [review]
LGTM, as Guiseppe mentioned, the description of the filesystem/sqlite storage backend is inaccurate right now. But otherwise this looks fine

File: docs/index.md (hunk 1 above)
Comment (jecluis) [reply docs/index.md]: I would drop it completely. Or simply say that we consume a filesystem provided by a PV, nothing else. I wouldn't mention metadata here because storing metadata is not a feature, but a necessity.

**Review by jecluis — CHANGES_REQUESTED** [review]

File: docs/index.md [hunk 2]
```diff
@@ -1,29 +1,51 @@
...
+- Ideal for small-scale deployments/Edge
+- Lightweight, simple User Experience
+- Simple: Storage/replication handled by a PV ([Longhorn][5])
+- Designed to integrate with Rancher's product catalog
+- Open source licensing
+
+[1]: https://github.com/aquarist-labs/ceph
+[2]: https://github.com/aquarist-labs/s3gw-tools
+[3]: https://github.com/aquarist-labs/s3gw-charts
+[4]:https://github.com/aquarist-labs/s3gw-ui
```
Comment (votdev) [inline docs/index.md:49]:
```suggestion
[4]: https://github.com/aquarist-labs/s3gw-ui
```

File: docs/index.md [hunk 3]
```diff
@@ -1,29 +1,51 @@
...
-the Ceph stack (no RADOS).
-
-## Summary
-
-In a nutshell, this project provides the required infrastructure to build a container
-able to run on a kubernetes cluster, providing S3-compatible endpoints to
-Kubernetes applications.
+## About the project
+
+The s3gw project is split into 4 work streams: [rgw/sfs-Ceph][1] (backend),
```
Comment (votdev) [inline docs/index.md:5]: Everything else is written in lower case in the next line, so we should do it here also.

```suggestion
The s3gw project is split into 4 work streams: [rgw/sfs-ceph][1] (backend),
```

File: docs/index.md [hunk 4]
```diff
@@ -1,29 +1,51 @@
...
+- K3S/Edge
+- SAP Data Intelligence
+
+### Value proposition
+
+- Ideal for small-scale deployments/Edge
+- Lightweight, simple User Experience
+- Simple: Storage/replication handled by a PV ([Longhorn][5])
+- Designed to integrate with Rancher's product catalog
+- Open source licensing
```
Comment (votdev) [inline docs/index.md:44]: Mention the license model here.

**Review by votdev — CHANGES_REQUESTED** [review]

File: docs/index.md (hunk 1 above)
Comment (jhmarina) [reply docs/index.md]: ok done

**Review by giubacc — APPROVED** [review]
